apps/meteor/client/components/message/variants/RoomMessage.tsx

```diff
@@ -62,6 +62,31 @@ const RoomMessage = ({
 
 	const messageRef = useJumpToMessage(message._id);
 
+
+	const handleRightClick = async (e: React.MouseEvent<HTMLDivElement>) => {
+        e.preventDefault();
+
+       // Check if the click target is an interactive element
+       const target = e.target as HTMLElement;
+       const isInteractiveElement = target.closest('a, button, [role="button"]');
+       if (isInteractiveElement) {
+        // Allow default right-click behavior for interactive elements
+        return;
+}
+		
+       const moreButton = (e.currentTarget as HTMLElement).querySelector('[aria-label^="More options"]') as HTMLElement;
```

Comment:
I don't think this approach is good because of a few reasons, first one:
- This would brake if we change the text of the aria-label
- If we decide for some reason to change ever so slightly the HTML this feature won't work

I think the better approach would be use it the same way we use when we click in the triple dots